genome_grist/conf/Snakefile

```diff
@@ -208,79 +208,74 @@ rule zip:
 # download SRA IDs.
 rule download_sra_wc:
     output:
-        r1 = protected(outdir + "/raw/{sample}_1.fastq.gz"),
-        r2 = protected(outdir + "/raw/{sample}_2.fastq.gz"),
+        r1  = protected(outdir + "/raw/{sample}_1.fastq.gz"),
+        r2  = protected(outdir + "/raw/{sample}_2.fastq.gz"),
         unp = protected(outdir + "/raw/{sample}_unpaired.fastq.gz"),
         temp_dir = temp(directory(f"{base_tempdir}/{{sample}}.d")),
+    params:
+        temp_r1 = lambda w: f"{base_tempdir}/{w.sample}.d/{w.sample}_1.fastq",
```

Comment:
why do you need to use lambda here? can't you use wildcards directly? I guess this works but seems overly complicated.

I'm thinking something like:

```
temp_r1 = f"{base_tempdir}/{{wildcards.sample}}.d/{{wildcards.sample}}_1.fastq"
```

Reply:
You can't access wildcards in params without lambda, right? But we could make them temporary outputs

Reply:
I'm not sure if this had been updated in recent versions of snakemake but yes, historically you could not use wildcards in params without lambda bc of the order of execution of the DAG